candidate/models.py

```diff
@@ -0,0 +1,93 @@
+from __future__ import unicode_literals
+from django.db import models
+from django.utils.encoding import python_2_unicode_compatible
+
+from ballot_measure.models import BallotMeasureChoice
+
+
+class SocialMediaModel(models.Model):
+    class Meta:
+        abstract = True
+
+    photo_url = models.ImageField(blank=True)
+    website_url = models.URLField(
+        help_text='URL for the official website.', blank=True)
+    facebook_url = models.URLField(
+        help_text='URL for the Facebook page.', blank=True)
+    twitter_url = models.URLField(
+        help_text='URL for the Twitter page.', blank=True)
+
+
+@python_2_unicode_compatible
+class Party(SocialMediaModel):
+    """
+    """
+    name = models.CharField(
+        max_length=255, help_text='The party name.')
+
+    def __str__(self):
+        return "The %s party" % self.name
+
+
+@python_2_unicode_compatible
+class Person(SocialMediaModel):
+    """
+    """
+    first_name = models.CharField(
+        max_length=255, help_text='The candidate\'s first name.')
+    middle_name = models.CharField(
+        max_length=255, blank=True, null=True,
+        help_text='The candidate\'s middle name.')
+    last_name = models.CharField(
+        max_length=255, blank=True, null=True,
+        help_text='The candidate\'s last name.')
+    party = models.ForeignKey('Party', blank=True, null=True)
+
+    def __str__(self):
+        return "%s, %s%s" % (
+            self.last_name, self.first_name,
+            '' if self.middle_name is None else ' %s.' % self.middle_name[0])
+
+
+@python_2_unicode_compatible
+class Office(models.Model):
+    """
+    """
+    name = models.CharField(
+        max_length=255, help_text='The office name.')
+    description = models.CharField(
+        max_length=1024, help_text='The office description.')
+    locality = models.ForeignKey('locality.Locality')
+
+    def __str__(self):
+        return "Office for %s in %s" % (
+            self.name, self.locality.dereference())
+
+
+@python_2_unicode_compatible
+class Election(models.Model):
```

Comment:
I'd prefer not to use the word "Election" to specifically mean the election of a person. I was trying to use the word Election to be the event when citizens vote.

I would suggest using the name "OfficeElection", "ElectionOffice" or "CandidateOffice", because it's very much about an Office up for election. However, I'm a little unclear on what the distinction is between this model and a BallotMeasure, since I'm under the impression an office up for election would be a BallotMeasure.